README.md

```diff
@@ -84,6 +86,70 @@ or have `InfoDialog` in their behavior text.
 Any option after the double dash (`--`) gets sent to `mocha`. Check out
 [mocha's usage](https://mochajs.org/#usage) for ideas.
 
+### Flow
+
+There is limited support for using [Flow](https://flowtype.org/)
+to check for problems in the source code.
+
+To check for Flow issues during development while you edit files, run:
+
+    npm run flow:dev
+
+If you are new to working with Flow, here are some tips:
+* Check out the [getting started](https://flow.org/en/docs/getting-started/) guide.
+* Read through the [web-ext guide](https://github.com/mozilla/web-ext/blob/master/CONTRIBUTING.md#check-for-flow-errors)
+  for hints on how to solve common Flow errors.
+
+To add flow coverage to a source file, put a `/* @flow */` comment at the top.
+The more source files you can opt into Flow, the better.
+
+Here is our Flow manifesto:
+
+* We use Flow to **declare the intention of our code** and help others
```

Comment:
These docs are really awesome now, thanks so much for doing another pass. Super easy to grok and the links are great 👍